sbt-dump-event-schema/src/main/scala/org/virtuslab/ash/DumpEventSchema.scala

```diff
@@ -1,11 +1,8 @@
 package org.virtuslab.ash
 
-import scala.language.postfixOps
-
 import better.files._
 import sbt.Keys._
-import sbt.{File => JFile, _}
```

Comment:
This was the reason `-imports` flag was specified, didn't know It could be solved like that